cns/restserver/restserver.go

```diff
@@ -48,15 +48,19 @@ const (
 // HTTPRestService represents http listener for CNS - Container Networking Service.
 type HTTPRestService struct {
 	*cns.Service
-	dockerClient     *dockerclient.DockerClient
-	imdsClient       *imdsclient.ImdsClient
-	ipamClient       *ipamclient.IpamClient
-	networkContainer *networkcontainers.NetworkContainers
-	routingTable     *routes.RoutingTable
-	store            store.KeyValueStore
-	state            *httpRestServiceState
-	lock             sync.Mutex
-	dncPartitionKey  string
+	dockerClient                 *dockerclient.DockerClient
+	imdsClient                   *imdsclient.ImdsClient
+	ipamClient                   *ipamclient.IpamClient
+	networkContainer             *networkcontainers.NetworkContainers
+	PodIPIDByOrchestratorContext map[string]string                     // OrchestratorContext is key and value is NetworkContainerID.
```

Comment:
I thought the value was the allocation uuid?

Reply:
Yep, the comment is wrong, will fix